<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server</class>
 <widget class="QWidget" name="Server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>本地端口号</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edit_localPort">
      <property name="inputMask">
       <string>00000</string>
      </property>
      <property name="text">
       <string>8888</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_listen">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>监听</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>201</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,8">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>本地IP地址</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="combo_ipList"/>
    </item>
   </layout>
  </widget>
  <widget class="QTextBrowser" name="text_chat">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>831</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_onlineList">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>70</y>
     <width>160</width>
     <height>690</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_italic">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>560</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/italic.png</normaloff>:/images/italic.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_bold">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>560</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/bold.png</normaloff>:/images/bold.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_fontSize">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>560</y>
     <width>72</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{combobox-popup:0;}</string>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="maxVisibleItems">
    <number>10</number>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_saveRecord">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>560</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
  </widget>
  <widget class="QFontComboBox" name="fontComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>203</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_clearRecord">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>560</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_underLine">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>560</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/under.png</normaloff>:/images/under.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_monitor">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>20</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>视频监控</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>831</width>
     <height>161</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPlainTextEdit" name="edit_send">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>831</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_send">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>130</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_userManager">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>20</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>用户管理</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
